src/Microsoft.Extensions.Configuration.FileExtensions/FileConfigurationProvider.cs

```diff
@@ -26,6 +26,7 @@ public FileConfigurationProvider(FileConfigurationSource source)
                 throw new ArgumentNullException(nameof(source));
             }
             Source = source;
+            OnLoadException = source.OnLoadException;
```

Comment:
Make `OnLoadException` property virtual and have the one on `FileConfigurationProvider` access `source.OnLoadException`?